VTS/Core/CoreVTSPlugin.cs

```diff
@@ -65,7 +65,7 @@ public void Dispose() {
 		public void Initialize(IWebSocket webSocket, IJsonUtility jsonUtility, ITokenStorage tokenStorage, Action onConnect, Action onDisconnect, Action<VTSErrorData> onError) {
```

Comment:
Honestly, we should probably just move these first three parameters to a constructor, leaving only the three callback params in the `Initialize` signature.

I wouldn't worry too much about changing the IVTSWebSocket interface, it's vestigial from when there used to be a Unity component version of the Socket (which processed responses in the Unity update loop, instead of in a worker thread like it does now). 

No one is actually making their own implementation that this kind of change would break.